docs/plus/cronjob.md

```diff
@@ -0,0 +1,178 @@
+# CronJob
+
+CronJob resource is responsible for creating recurring [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/). The job recurrence is determined by a [Schedule](https://github.com/cdk8s-team/cdk8s-core/blob/2.x/src/schedule.ts).
+
+CronJob is similar to a [job](https://cdk8s.io/docs/latest/plus/job/) but it is suitable when there is a need to run a job indefinitely following a schedule. These repetitive jobs can be utilized for recurring tasks such as backing up a database, pinging a server for health checks, creating snapshots of systems and much more.
+
+!!! tip "" 
+     [API Reference](../reference/cdk8s-plus-24/typescript.md#cronjob)
+
+## Creating a CronJob
+
+You can create a cronjob using a cdk8s+ CronJob construct. The library provides a set of defaults, so to provision a CronJob instance you just need to provide it with the job's properties.
+
+### Creating a CronJob using CronJob construct:
+
+```typescript
+import { App, Chart, Duration, Schedule } from 'cdk8s';
+import * as kplus from 'cdk8s-plus-24';
+
+const app = new App();
+const chart = new Chart(app, 'Chart');
+
+// lets setup a cron job here which backups mysql database
+// by default cronjob would schedule job to run every minute
+const backupCronJob = new k8splus.CronJob(this, 'BackupCronJob', {
+  jobProperties: {
+    activeDeadline: Duration.minutes(5),
+    backoffLimit: 5,
+    },
+  containers: [
+      {
+        image: 'databack/mysql-backup',
+      }
+    ]
+});
+```
+
```

Comment:
This is where I would add a paragraph mentioning the important default values that we set:

*This will create a `CronJob` resource that runs the `databack/mysql-backup` every X minutes with an active deadline of 5 minutes and a backoff limit of 5*. To customize the properties, see [Link to API reference].